dpdata/cp2k/output.py

```diff
@@ -225,6 +251,7 @@ def handle_single_log_frame(self, lines):
         info_dict['cells'] = np.asarray([self.cell]).astype('float32')
         info_dict['energies'] = np.asarray([energy]).astype('float32')
         info_dict['forces'] = np.asarray([forces_list]).astype('float32')
+        if(virial != None ): info_dict['virials'] = np.asarray([virial]).astype('float32')
```

Comment:
Per PEP-8 style, `!= None` should be written as `is not None`. Also, do not write if statement as one line.